solutions/src/test/java/ru/sberbank/school/task10/FixedThreadPoolTest.java

```diff
@@ -0,0 +1,45 @@
+package ru.sberbank.school.task10;
+
+import org.junit.jupiter.api.*;
+
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class FixedThreadPoolTest {
+    static ThreadPool threadPool;
+    static AtomicInteger count;
+
+    @BeforeEach
+    public void init() {
+        threadPool = new FixedThreadPool(2);
+        threadPool.start();
+        count = new AtomicInteger(0);
+    }
+
+    @AfterEach
+    public void stop() {
+        threadPool.stopNow();
+    }
+
+    @Test
+    public void checkIllegalConstructorArgument() {
+        Assertions.assertThrows(IllegalArgumentException.class, () -> new FixedThreadPool(0));
+    }
+
+    @Test
+    public void checkCountOfExecutedTask() {
+        for (int i = 0; i < 20; i++) {
+            threadPool.execute(() -> {
+                    count.incrementAndGet();
```

Comment:
&#39;block&#39; child has incorrect indentation level 20, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-478675783>*